try-it.md

```diff
@@ -118,7 +118,6 @@ hosts. You can see the VMs using `virsh`.
 $ sudo virsh list
  Id    Name                           State
 ----------------------------------------------------
- 6     minikube                       running
```

Comment:
Why is this removed? We still use minikube for CentOS based Metal3

Reply:
I have installed the dev-env with CentOS images but this minikube vm still didn't come up, from the source it looks to me that it only comes up if I explicitly set the EPHEMERAL_CLUSTER environment variable to have the value "minikube". I will document both case when EPHEMERAL_CLUSTER is set to "minikube" and when it is set to "kind".

Reply:
Did you have CentOS OS on the host VM where metal3-dev-env was running or was CentOS used for your target VMs?

Reply:
Host VM was Ubuntu 20 and I used CentOS for target VMs.

Reply:
ok that explains :) So there two different OS that we test metal3-dev-env on
1. Host VM/Server on CentOS, while target can be Ubuntu or CentOS, etc.
2. Host VM/Server on Ubuntu, while target can be Ubuntu or CentOS, etc.

The way we run k8s clusters for above two scenarios is different. For CentOS we use minikube cluster, for Ubuntu Kind cluster. As such, when your host is CentOS, you should be able to see three libvirt VMs. Since you are using Ubuntu, the source k8s cluster was created via Kind, which is why you don't see minikube VM. To control what tool should be used for creating source k8s cluster, we use EPHEMRAL_CLUSTER variable, which is minikube by default for CentOS and kind for Ubuntu.

Reply:
Thanks for the clarification, I have incorporated your explanation to the try-it.md with a new commit, I think it could be useful for beginners.